### ISSUE TITLE
How are you authenticating with api?

### ISSUE THREAD

**Issue by shaansheikh** [leMaik/moneylover-cli#2]
How did you do this? As far as I know, Moneylover is not giving out api keys yet. Did you manage to somehow obtain one? Looking through the code, I can't really understand how you're authenticating

**Comment by leMaik** [comment]
Oh wow, someone uses this tool? :tada: 
TL;DR: No API key is used. This client just behaves like the browser would, sending the same requests.

### How it works for OAuth (Facebook or Google login, but also works for username+password):
I'm authenticating by opening Chrome/Chromium and using the debug protocol to grab the access token they issue. They put them into the URL during the redirect. It's all [in this file](https://github.com/leMaik/moneylover-cli/blob/master/src/chromeLogin.js).

Another way is to grab that manually from the Networking console during login in your default browser and copy/pasting that into the `moneylover login --jwt ...` command, [as seen here](https://github.com/leMaik/moneylover-cli#using-facebook-or-google-without-google-chrome-installed).

### How it works with username/password login:
Far less exciting, it's [just a http request](https://github.com/leMaik/moneylover-cli/blob/master/src/moneylover.js#L34).
